packages/host/app/components/operator-mode/container.gts

```diff
@@ -136,6 +139,65 @@ export default class OperatorModeContainer extends Component<Signature> {
     return await this.operatorModeStateService.constructRecentCards();
   });
 
+  @action async onCommand(command: any) {
+    // apply patch
+    if (command.type == 'patch') {
+      for (let item of this.allStackItems) {
+        if ('card' in item && item.card.id == command.id) {
+          await this.setFieldValues(item.card, command.patch.attributes);
+          return await this.save(item);
+        }
+      }
+    }
+  }
+
+  @action async onPreviewCommand(command: any) {
+    // apply patch
+    if (command.type == 'patch') {
+      for (let item of this.allStackItems) {
+        if ('card' in item && item.card.id == command.id) {
+          await this.saveCardFieldValues(item.card);
+          await this.setFieldValues(item.card, command.patch.attributes);
+        }
+      }
+    }
+  }
+
+  @action async onCancelPreviewCommand(command: any) {
+    // reset field values
+    if (command.type == 'patch') {
+      for (let item of this.allStackItems) {
+        if ('card' in item && item.card.id == command.id) {
+          return await this.cancel(item);
```

Comment:
hmmm, i thought we got rid of `cancel` on the main branch. there is no more cancelling of changes to cards since we have autosave now.